### ISSUE TITLE
Library doesn't export document

### ISSUE THREAD

**Issue by pyrokinetiq** [christophehurpeau/html-document#1]
`index.js` has no `module.exports` rendering this module is useless in node.js

**Comment by christophehurpeau** [comment]
`global` makes document useable everywere : https://nodejs.org/api/globals.html#globals_global
You can then use nodejs as if there were only one document for the all application, depending on your needs it might not be what you want.
It is enough for instance if you only create document fragments like `var myfragment = document.createDocumentFragment()` and play with them

If you want to instantiate the document yourself, you can do it like this:

```
var Document = require("html-document/lib/Document").Document;
var document = new Document();
```

Hope I'm clear enough, don't hesitate if have questions

**Comment by christophehurpeau** [comment]
Did you resolve your issue ?

**Comment by pyrokinetiq** [comment]
While this method "works" it does not follow standard npm/node.js module signature. To be used in node.js it should use the `module.exports` assignment rather than polluting global namespace.

If a user wishes to make the document global, then the module should still use `module.exports` and the user can simply do `global.document = require('html-document');`

This way usage of this module is consistent with all other npm modules.

**Comment by christophehurpeau** [comment]
Closed with 104bccf2aa6c26075065522b5e2406f376e871a1
